src/main/resources/reference.conf

```diff
@@ -69,6 +69,10 @@ videobridge {
     # set its size (in terms of millis) to be 16ms.
     # With interbridge last-n the load would be much smaller.
     recv-queue-size=1024
+    # The size of the outgoing octo queue.  This is a tentacle level queue and
+    # it's where we temporarily store packets to be sent to remote bridges.
+    # TODO: tune this.
+    send-queue-size=1024
```

Comment:
A quick n' dirty calculation of the queue depth would be nice (but not mandatory).